sdk/chaos/arm-chaos/package.json

```diff
@@ -47,7 +47,7 @@
     "cross-env": "^7.0.2",
     "@types/node": "^16.0.0",
     "@azure/dev-tool": "^1.0.0",
-    "@azure/arm-cosmosdb": "16.0.0-beta.7",
+    "@azure/arm-cosmosdb": "16.0.0-beta.6",
```

Comment:
Is there a test in the chaos library that relies on comosdb ? is it possible to upgrade to the latest ? @kazrael2119

Reply:
yes, we should run chaos test cases after generating cosmosdb